api/schema/types/ClientType.js

```diff
@@ -16,11 +16,19 @@ module.exports = gql`
         ): String
     }
 
-    input ClientInput {
+    input ClientCreateInput {
         is_active: Boolean,
         name: String,
         email: String,
         currency: String,
+        contributor_id: Int
+    }
+
+    input ClientInput {
+        is_active: Boolean,
+        name: String,
+        email: String,
+        currency: String
```

Comment:
I think we don't need 2 separate input types especially since there are no strictly required`!` fields

Reply:
The CinetCreateInput actually should be requires, so I will leave both input objects but one of them will be required